it/services/ApplicationAssessmentServiceSpec.scala

```diff
@@ -69,7 +69,7 @@ class ApplicationAssessmentServiceSpec extends MongoRepositorySpec with MockitoS
   val DebugTestNameAppId: Option[String] = None
 
   "Assessment Centre Passmark Service" should {
-    "for each test in the path evaluate scores" in new WithApplication {
+    "for each test in the path evaluate scores" ignore new WithApplication {
```

Comment:
the functionality is not currently being used, and it would need some fixing to work with the ScalaTest play helpers, rather than the specs 2 WithApplication, which we should not be using.  The format of these tests are also fairly unconventional. TL;DR; I put ignore because it may need fixing in the future, but is a problem now if we don't disable.